vue3-ts-quasar/src/composables/github/types/Gist.ts

```diff
@@ -0,0 +1,6 @@
+export default interface GistItem {
+  id: string;
+  description?: string | null;
+  name: string;
+  url: string;
+}
```

Comment:
Prefer named exports in types since that means we can have multiple related types in one file.
```ts
export interface GistItem
```